docs/subnet.md

```diff
@@ -10,7 +10,12 @@ We provide instructions for running both a [simple single-validator subnet](#run
 
 ### Exporting wallet keys
 
-In order to run a validator in a subnet, we'll need a set of keys to handle that validator. To export the validator key from a wallet that may live in another network into a file (like the wallet address we are using in the rootnet), we can use the following Lotus command:
+In order to run a validator in a subnet, we'll need a set of keys to handle that validator. To export the validator key from your agent you need to run: 
+```bash
+./ipc-agent/bin/ipc-agent wallet export --address=<address-to-export> --output=<output file>
+```
+
+To export a wallet that may live in another network into a file (like the wallet address we are using in the rootnet), we can use the following Lotus command:
```

Comment:
Just to be clear, the idea here is that you're providing instructions to export both from ipc-agent (if using identity management) or from raw eudico (for keys not held by the agent). Is that it?

Reply:
Yes, that was the idea, just in case someone has a set of keys on a lotus peer that they want to use in the agent.

Reply:
I rephrased the paragraph a bit to make a bit clearer. Thank you for raising it.